samples/test/authViewTutorial.test.js

```diff
@@ -36,7 +36,8 @@ const sharedViewId = generateUuid();
 
 const bigquery = new BigQuery();
 
-describe('Authorized View Tutorial', () => {
+describe('Authorized View Tutorial', function () {
+  this.retries(3);
```

Comment:
I think doing this way, it will retry the whole `describe` section, which can be quite expensive to re run. I ran some tests here and seems like we can retry on a per spec basis.

```
describe('Authorized View Tutorial', () => { // Back to arrow function
  beforeEach(function(){
    console.log("this:", this.currentTest.title)
    this.currentTest.retries(2); // will try 3 times - n+1    
  })
```

Then I changed one of the tests to fail, to check for the retry behavior:

```
  it('should query stackoverflow', async () => {
    const output = execSync('node queryStackOverflow.js');
    assert.match(output, /Query Results:/);
    assert.match(output, /views/);
    assert.match(0,1)
  }); // Force failure
  ```